### ISSUE TITLE
Any way to use VSCode breakpoints along with this fantastic debugger?

### ISSUE THREAD

**Issue by jhalborg** [jhen0409/react-native-debugger#206]
Thanks for a great lib!

I may have missed something, but is there any way to use in-IDE breakpoints in VSCode when using this debugger? As I understand it, there can only be one process on the same port at a time, so I'm not sure. I've tried 'attach to packager', but get told that another debugger is already attached. For a while I've been using remotedev.io for redux inspection, and then had the ability to do breakpoints from VSCode, but seeing as the former is unstable AF, I thought I'd at least ask :-)

For me, that's literally the only thing missing from making this the perfect debug experience.

**Comment by jhen0409** [comment]
I think it's possible if VSCode support WebWorker debugging (https://github.com/Microsoft/vscode-chrome-debug-core/issues/14)

**Comment by spacesuitdiver** [comment]
@jhalborg any luck getting vs code breakpoints to work? I went full circle, trying to use `remotedev-rn-debugger` to work with redux in vs code but ultimately realized life without the element inspector and network tab is terrible. I think the best development experience is to simply use this app and incorporate breakpoints with vs code somehow.

**Comment by jhalborg** [comment]
Nope, never did - I've accepted my fate without the network tab, and use the emulator element inspector
